packages/babel-parser/src/plugins/flow/index.js

```diff
@@ -1823,18 +1819,14 @@ export default (superClass: Class<Parser>): Class<Parser> =>
     // interfaces and enums
     parseStatement(context: ?string, topLevel?: boolean): N.Statement {
       // strict mode handling of `interface` since it's a reserved word
-      if (
-        this.state.strict &&
-        this.match(tt.name) &&
-        this.state.value === "interface"
-      ) {
+      if (this.state.strict && this.isContextual(tt._interface)) {
```

Comment:
This is a bug fix:
```js
interf\u{61}ce A {}
```
is [invalid Flow code](https://flow.org/try/#0JYOwLgpgTgZgOgVwN4DYCMBfAxhABAQVyQyA).